src/Microsoft.Extensions.Configuration/IncludeConfigurationProvider.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.Configuration
+{
+    public class IncludeConfigurationProvider : ConfigurationProvider
```

Comment:
Yeah these are easy to change, for me I probably rank them.  

1. Include
2. Copy
3. Import

I think its clearer for the name to exactly match the extension method that adds it, since these classes aren't used directly